config/locales/components/en.yml

```diff
@@ -19,6 +19,8 @@ en:
       enabled_alert: You have enabled JavaScript in your browser.
       next_step: Please refresh this page once you have enabled JavaScript in your
         browser.
+    login_button:
+      image_alt: Login.gov logo
```

Comment:
This appears to be unused, I suspect this is one of the reasons for the build failures.